bdk-ffi/src/bitcoin.rs

```diff
@@ -675,6 +682,7 @@ impl_hash_like!(TxMerkleNode, BitcoinDoubleSha256Hash);
 
 #[cfg(test)]
 mod tests {
+    use super::Psbt;
```

Comment:
nit: are we able to use this `use crate::bitcoin::Psbt;` instead just to keep it consistent with the

```
use crate::bitcoin::Network;
use crate::bitcoin::Address;
```

below it

Reply:
thanks